test/e2e/node/pre_stop.go

```diff
@@ -202,7 +202,7 @@ var _ = SIGDescribe("PreStop", func() {
 		framework.ExpectNoError(err, "failed to delete pod")
 
 		//wait up to graceful termination period seconds
```

Comment:
just a small suggestion - perhaps we can explain why it is 15 seconds in a comment here.